.github/workflows/BundleStaticLibs.yml

```diff
@@ -124,7 +124,6 @@ jobs:
 
       - uses: ./.github/actions/build_extensions
         with:
-          deploy_as: windows_amd64_mingw
```

Comment:
This in particular was problematic, since here it would deploy extensions (possibly overriding the one built in R.yml) with chances that those workflows gets out-of-sync.